### DISCUSSION TITLE
Lodestar Standup 2024-10 (March 12, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#6510] [Standups]
### Meeting Info
[Mar 12, 2024, 14:00-15:00 UTC](https://savvytime.com/converter/utc-to-germany-berlin-united-kingdom-london-ny-new-york-city-ca-san-francisco-china-shanghai-japan-tokyo-australia-sydney/mar-12-2024/2pm)
Zoom: shared on #lodestar-general ChainSafe Discord channel at 14:00 UTC

### Agenda
- Planning Items
  - v1.17.0 Release
  - Final Deneb Upgrades
- Team Updates
- Open Discussion/Closing Remarks

### Instructions
Please post here any topics to be added for Standup discussions. Examples of agenda items are issues, discussions, proposals which require feedback, consensus from the team and/or community members.

**Comment by philknows** [comment]
@ensi321 brought up a good discussion point to start thinking about potential opportunities for mentorship with the next cohort of Ethereum Protocol fellows. Previously, it was very informal and engineers on the team who wanted to dedicate some time to helping a fellow could do so, but there was no "formal plan" to have them take on a 4-month project to contribute to Lodestar development. This time around, it would be good to think of what medium-sized project (ideally 4 month duration of estimated work) that we can have a contributor complete with our guidance. This will ideally benefit not just protocol development on Ethereum, but also hopefully have some contributed code that would be useful for Lodestar.

**Comment by ensi321** [comment]
I would also like to bring up #6518 to see if anyone has any opinion to express as I've seen some discussions in the issue. 

> It could also be an opportunity to write some guidelines detailing which tests are expected to be added depending on the feature being introduced.

@jeluard made a good point that we might also want to establish a convention/practice regarding what level of testing is needed when introducing a new feature

**Comment by philknows** [comment]
We should also discuss bumping `--max-old-space` to 16384 as we get into post-Deneb, eventually raising our max peer limit and for more guaranteed stability. This is in no way to band-aid memory optimizations but rather to ensure by default people do not have their heap memory kill Lodestar due to insufficient memory. 8GB machines for Ethereum nodes/staking is starting to become rare.

**Comment by twoeths** [comment]
I'd like to release new version of ssz https://github.com/ChainSafe/ssz/pull/351

**Comment by philknows** [comment]
Closing for #6553 
Notes located here: https://github.com/ChainSafe/lodestar/wiki/Lodestar-Planning-&-Standup-Meetings#march-12-2024-planning-and-standup-meeting
